### PR TITLE
PR: Eliminate redundant links from sidebar, add standard links to readme and fix minor layout bugs

### PR DESCRIPTION
**Pull request by CAM-Gerlach**
### Pull Request Checklist

* [x] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [x] Based your PR on the latest version of the correct branch (master or 3.x)
* [x] Checked your writing carefully for correct English spelling, grammar, etc
* [x] Described your changes and the motivation for them below
* [x] ~~Noted what issue(s) this pull request resolves, creating one if needed~~

## Description of Changes

Currently, our Facebook and especially Twitter links aren't really linked from anywhere but our OpenCollective to my knowledge, and correspondingly have very low SEO rankings and are very hard to find. Therefore, this PR adds them to our readme and sidebar, and fixes a very minor issue with the ToC link layout (along the lines of #27 ).

### Issue(s) Resolved

Minor meta-task; not needed.

### CODE REVIEW

**Comment by ccordoba12** [comment]
I don't agree with this. I think we should add these links to our footer instead.

**Comment by CAM-Gerlach** [comment]
> I think we should add these links to our footer instead.

Okay, sure—that makes more sense than cramming yet more stuff into the sidebar. Do you mean the default docs footer, or a custom footer element based on a minimized version of the one on our main site, like we did for the banner? Also, see my comment on spyder-ide/website-spyder#113 .

**Comment by CAM-Gerlach** [comment]
@ccordoba12 I'll just go ahead and put them in the current footer, and if you'd like something different, let me know.

**Comment by CAM-Gerlach** [comment]
Done, after some fumbling around. There's apparently no provision to add it as a theme setting, so I had to teach myself some more html and css and do it manually. I moved a few other social media/connect with us links as well, to further reduce clutter in the sidebar and keep it thematically focused on static help resources, with the footer being the interactive help/contact options (moving Google Groups and Gitter as well, and adding OC there). Here's how it looks:

image

I'm a total novice with all this, so let me know what you want changed.

**Comment by CAM-Gerlach** [comment]
> this looks nice! We should use the same footer here and in our other web pages.

Okay, so do you mean using _this_ footer on the mainpage/blog? In any case, I'm definitely not a huge fan of the existing one (at least the text layout); it takes up way too much space and the line break at the very end of the top line is really awful. If so, I would imagine we'd want to use the top "Connect with us" line verbatim, and then have the bottom one (or likely multiple) be the copyright/etc text from the current website footer instead of the docs ones (while we're at it, some rights reserved can be removed to save a little space; it doesn't add anything when the exact license is specified and linked).

> custom right is to use a dark gray background for the footer, to differentiate from the rest of the page.

Not sure I understand what you mean here...

**Comment by CAM-Gerlach** [comment]
@ccordoba12 ?

**Comment by ccordoba12** [comment]
Sorry for the delay. I assigned @dalthviz to work on a new footer this week.

**Comment by CAM-Gerlach** [comment]
Oh, excellent—thanks for the update. I assume that'll replace this one as well as the website and blog?

**Comment by ccordoba12** [comment]
Yep, that's the idea.

**Comment by CAM-Gerlach** [comment]
Okay, so should I remove the one I have in this PR right now, so they don't conflict (merge and otherwise)? Then this could presumably get merged, since otherwise it doesn't depend on the footer.

**Comment by ccordoba12** [comment]
Yes, please do.

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Thanks, done. 

So much for that, heh...but hey, I managed to learn some very basic HTML and CSS along the way.

File: doc/conf.py [hunk 1]
```diff
@@ -126,14 +126,11 @@
     'travis_button': False,
     'codecov_button': False,
     'extra_nav_links': {
```
Comment (ccordoba12) [inline doc/conf.py]: Let's remove all these extra links in this PR. We'll put them all in the footer.

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Updated to fix conflicts and ensure it is a superset of PR #62 ; namely, eliminate the Github, Download, and Donate links and the translation sectioned, and tweaked the wiki link to point to the index page instead of the dev page. 

So, aside from eliminating the now redundant links (and at least two others #62 missed, Gitter and Google Groups, which are now in the footer), this PR makes a few related minor tweaks:

* Standardizes the readme with the same links (FB, Twitter, etc) as the others
* Fixes two minor related link/list layout bugs along the lines of #27 
* Minor trailing-space cleanup in layout.html

which should be fairly straightforward and uncontroversial.

**Pull request merged**